samza-core/src/main/java/org/apache/samza/storage/TaskSideInputStorageManager.java

```diff
@@ -120,35 +76,32 @@ public void init() {
     Map<SystemStreamPartition, String> fileOffsets = getFileOffsets();
     LOG.info("File offsets for the task {}: {}", taskName, fileOffsets);
 
```

Comment:
where is this getting used? or is it just for logging purposes?

Reply:
Good point. Nope, not used here. I'll remove this and add the log statement into the handler's init method.